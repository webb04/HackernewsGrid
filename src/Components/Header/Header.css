.Header {
    background: rgb(255, 102, 0);
    color: white;
    padding: 20px;
}

.Header-title {
    margin: 0;
    text-align: left;
}