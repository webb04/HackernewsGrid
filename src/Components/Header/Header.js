import React, { Component } from 'react';
import './Header.css';

class Header extends Component {
    render() {
        return (
            <header className="Header">
                <h1 className="Header-title">Hacker News</h1>
            </header>
        )
    }
}

export default Header;