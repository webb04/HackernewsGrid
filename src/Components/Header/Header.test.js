import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import renderer from 'react-test-renderer';

it('Renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Renders correctly', () => {
    const tree = renderer
      .create(<Header />)
      .toJSON();
    expect(tree).toMatchSnapshot();
});
