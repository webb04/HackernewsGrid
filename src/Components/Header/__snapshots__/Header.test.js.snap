// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<header
  className="Header"
>
  <h1
    className="Header-title"
  >
    Hacker News
  </h1>
</header>
`;

exports[`renders correctly 1`] = `
<header
  className="Header"
>
  <h1
    className="Header-title"
  >
    Hacker News
  </h1>
</header>
`;
